chat/src/main/kotlin/net/thechance/chat/service/AttachmentStorageService.kt

```diff
@@ -58,11 +78,18 @@ class AttachmentStorageService(
     }
 
     private companion object {
-        val allowedMimeTypes = mapOf(
+        val allowedImageMimeTypes = mapOf(
             "image/jpeg" to "jpg",
             "image/jpg" to "jpg",
             "image/png" to "png",
             "image/webp" to "webp",
         )
+
+        val allowedAudioMimeTypes = mapOf(
+            "audio/mpeg" to "mp3",
+            "audio/wav" to "wav",
+            "audio/ogg" to "ogg",
+            "audio/mp4" to "m4a",
+        )
```

Comment:
```suggestion
        val allowedAudioMimeTypes = mapOf(
            "audio/mp3" to "mp3",
            "audio/wav" to "wav",
            "audio/ogg" to "ogg",
            "audio/m4a" to "m4a",
        )
```

are you sure it's not like this? 👀